### PR TITLE
Update impressions

### PR DESCRIPTION
**Pull request by weshaggard**
Update impression pixel links and the package index.

PTAL @scbedd

cc @Azure/azure-sdk-eng

### CODE REVIEW

**Comment by adxsdk6** [comment]
Can one of the admins verify this patch?

File: tools/azure-sdk-tools/packaging_tools/templates/README.rst [hunk 1]
```diff
@@ -30,4 +30,4 @@ If you encounter any bugs or have suggestions, please file an issue in the
 section of the project.
 
 
-.. image::  https://azure-sdk-impressions.azurewebsites.net/api/impressions/azure-sdk-for-python%2F{{package_name}}%2FREADME.png
+.. image::  https://azure-sdk-impressions.azurewebsites.net/api/impressions/azure-sdk-for-python%2Ftools%2Fazure-sdk-tools%2Fpackaging_tools%2Ftemplates%2FREADME.png
```
Comment (lmazuel) [inline tools/azure-sdk-tools/packaging_tools/templates/README.rst]: This one is a template, you want to keep `{{package_name}}`` in it.

File: packages.md [hunk 2]
```diff
@@ -128,14 +138,17 @@
...
 | [`azure-servicemanagement-legacy`]( sdk/core/azure-servicemanagement-legacy/setup.py )| [Readme](sdk/core/azure-servicemanagement-legacy/README.rst) | [Changelog](sdk/core/azure-servicemanagement-legacy/HISTORY.rst) | [PyPI]( https://pypi.org/project/azure-servicemanagement-legacy ) |
 | [`azure-storage-blob`]( sdk/storage/azure-storage-blob/setup.py )| [Readme](sdk/storage/azure-storage-blob/README.md) |  N/A  | [PyPI]( https://pypi.org/project/azure-storage-blob ) |
-| [`azure-storage-file-share`]( sdk/storage/azure-storage-file-share/setup.py )| [Readme](sdk/storage/azure-storage-file-share/README.md) |  N/A  | [PyPI]( https://pypi.org/project/azure-storage-file-share ) |
+| [`azure-storage-file-share`]( sdk/storage/azure-storage-file-share/setup.py )| [Readme](sdk/storage/azure-storage-file-share/README.md) |  N/A  |  N/A  |
 | [`azure-storage-queue`]( sdk/storage/azure-storage-queue/setup.py )| [Readme](sdk/storage/azure-storage-queue/README.md) |  N/A  | [PyPI]( https://pypi.org/project/azure-storage-queue ) |
+| [`pylint-guidelines-checker`]( scripts/pylint_custom_plugin/setup.py )| [Readme](scripts/pylint_custom_plugin/README.md) |  N/A  |  N/A  |
```
Comment (johanste) [inline packages.md]: Why do we need to include this package?

Comment (weshaggard) [reply packages.md]: We don't it was just in the repo :) I will clean that up.

File: tools/azure-sdk-tools/packaging_tools/templates/README.rst (hunk 1 above)
Comment (weshaggard) [reply tools/azure-sdk-tools/packaging_tools/templates/README.rst]: I will fix this.

**Comment by weshaggard** [comment]
Closing this PR as I've not been able to get back to this. If I get back to it I'll open another PR in the future.

**Pull request closed without merging**